arbos/util/storage_cache.go

```diff
@@ -67,6 +69,10 @@ func (s *storageCache) Flush() []storageCacheStores {
 			})
 		}
 	}
+	sortFunc := func(a, b storageCacheStores) int {
+		return bytes.Compare(a.Key.Bytes(), b.Key.Bytes())
```

Comment:
nit: `return a.Key.Cmp(b.Key)` would be a bit simpler

Reply:
Done.